app/controllers/pull-requests.js

```diff
@@ -46,15 +46,8 @@ export default class PullRequestsController extends Controller {
     return Array.from(uniqueUsers.values());
   }
 
-  sortUsers(users) {
-    return users.sort((a, b) => {
-      const value1 = a.handle.toLowerCase();
-      const value2 = b.handle.toLowerCase();
-
-      if (value1 > value2) return 1;
-      if (value1 < value2) return -1;
-      return 0;
-    });
+  listUsersInRandomOrder(users) {
+    return users.sort(() => Math.random() - 0.5);
```

Comment:
I think the article from javascript.info had mentioned that this algorithm has bias. It's a good start but, in a separate pull request, I think I would implement the Fisher-Yates shuffle (also mentioned in the article).